src/meson.build

```diff
@@ -11,6 +11,7 @@ snapd_desktop_integration = executable(
   'sdi-refresh-dialog.c',
   'sdi-refresh-monitor.c',
   'sdi-theme-monitor.c',
+  'sdi-user-session-manager.c',
```

Comment:
Doesn't sound like a manager, as there is no class just one helper method. Not sure of a better name but perhaps something like `sdi-graphical-session-helper.c`.

Reply:
You are right. Changing it.

Reply:
Done.